backend/server.js

```diff
@@ -1,41 +1,49 @@
-import express from 'express';
-import session from 'express-session';
-import cors from 'cors';
-import morgan from 'morgan';
-import { sequelize } from './database.js';
-import { User, Post } from './models/index.js';
-import userRoutes from './routes/users.js';
-import sequelizeStoreInit from 'connect-session-sequelize';
-
-const app = express();
-
-app.use(cors({
-  origin: 'http://localhost:3001',
-  credentials: true
-}));
-
-app.use(express.json());
-app.use(morgan())
-
-const SequelizeStore = sequelizeStoreInit(session.Store);
-const sessionStore = new SequelizeStore({
-  db: sequelize
-});
-
-app.use(
-    session({
-        secret: 'your-secret-key',
-        resave: false,
-        saveUninitialized: false,
-        store: sessionStore,
-        cookie: {
-            sameSite: 'false',
-            secure: false,
-            expires: new Date(Date.now() + (365 * 24 * 60 * 60 * 1000)) // 1 year in milliseconds
-        }
-    })
-);
-
-sessionStore.sync();
-
-app.use( userRoutes );
+// import express from 'express';
```

Comment:
If we're commenting out all of this code, should we just remove the file?

Reply:
I will remove it